<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentView</class>
 <widget class="QWidget" name="StudentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>556</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Hack Nerd Font</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Student View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/resources.qrc">
    <normaloff>:/resources/ict-university-461753.jpg</normaloff>:/resources/ict-university-461753.jpg</iconset>
  </property>
  <widget class="QTableWidget" name="StudentResultsTable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>661</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack Nerd Font</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutStudent">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EmailStudentResults_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>View Results</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Viewing...</string>
   </property>
   <property name="icon">
    <iconset resource="../Icons/resources.qrc">
     <normaloff>:/resources/check.png</normaloff>:/resources/check.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter Your Matricule:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="StudentMAtricule">
      <property name="placeholderText">
       <string>ICTU...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
